Classes/GHNSURL+Utils.m

```diff
@@ -62,86 +53,23 @@ + (NSArray *)gh_dictionaryToQueryArray:(NSDictionary *)queryDictionary sort:(BOO
     }
     
     if (!valueDescription) continue;
-    
-    if (encoded) key = [self gh_encodeComponent:key];
-    if (encoded) valueDescription = [self gh_encodeComponent:valueDescription];
-    [queryStrings addObject:[NSString stringWithFormat:@"%@=%@", key, valueDescription]];
+    NSURLQueryItem *item = [NSURLQueryItem queryItemWithName:key value:valueDescription];
```

Comment:
After much research, we do in fact need to preserve the old behavior – this is a breaking change that's incompatible with urllib, among others. Will migrate this back to the old behavior.